include/deal.II/base/exceptions.h

```diff
@@ -1130,6 +1130,25 @@ namespace StandardExceptions
 #define AssertIsFinite(number) Assert(dealii::numbers::is_finite(number), \
                                       dealii::ExcNumberNotFinite(std::complex<double>(number)))
 
+#ifdef DEAL_II_WITH_CUDA
+/**
+ * An assertion that checks that the error code produced by calling a CUDA
+ * routine is equal to cudaSuccess.
+ *
+ * @ingroup Exceptions
+ * @author Bruno Turcksin, 2016
+ */
+#define CudaAssert(error_code)                                                \
+  {                                                                           \
+    if (cudaSuccess != error_code)                                            \
```

Comment:
It is entirely equivalent, but would you mind writing this as
```
  if (error_code != cudaSuccess)
```
? :-) I find that easier to read, and it matches how we usually think of questions.